### ISSUE TITLE
Question:  does livestreamer support using a local m3u8 rather than a web-based m3u8?

### ISSUE THREAD

**Issue by streammonkey** [chrippa/livestreamer#1514]
I've tried everything I can think of but to no avail.  Is it possible for livestreamer to use a local .m3u8?  If not, is that something that could be added?

Thank you.

**Comment by ghost** [comment]
No, you need to setup a local server to do this.
For example you can download QuickPHP and change the root folder to the folder you have the m3u8, and then call it via livestreamer with http://LOCALIP:LOCALPORT/EXAMPLE.m3u8

**Comment by streammonkey** [comment]
Thanks.  It would be hugely beneficial to many people if a local read were added.  Not everyone is equipped to set up a local server.

**Comment by ghost** [comment]
Yes, this would be a great feature, but sadly the livestreamer Project is innactive now :(
But you can try requesting this in a fork called streamlink: https://github.com/streamlink/streamlink
